.gitignore

```diff
@@ -44,6 +44,12 @@ files/Aboot/boot0
 files/initramfs-tools/arista-convertfs
 files/initramfs-tools/union-mount
 
+# Subdirectories in src
+src/preload/gcovpreload/debian/*
+!src/preload/gcovpreload/debian/changelog 
+!src/preload/gcovpreload/debian/control
+!src/preload/gcovpreload/debian/rules 
+
```

Comment:
Please add the .gitignore in the src/preload/, not necessary to expose in the root .gitignore.

Reply:
updated